packages/extension-vscode/.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+{
+	"version": "0.2.0",
```

Comment:
We use [4 spaces for the indentation](https://github.com/webhintio/hint/blob/7e94f0bd8a8ae40c7fff7916febb917f376307b6/.editorconfig#L13).